internal/util/util.go

```diff
@@ -398,8 +398,11 @@ func GetVolumeSnapshotRestoreWithStatusData(restoreName string, PVCName string,
 		}
 
 		if len(vsrList.Items) > 0 {
-			if vsrList.Items[0].Status.Phase == "Failed" {
-				return false, errors.Errorf("volumesnapshotrestore %v has failed status", vsrList.Items[0].Name)
+			if vsrList.Items[0].Status.Phase != "" {
```

Comment:
Phase isn't a pointer tho?

Reply:
If there exists Item[0] then item[0].name would exists. I don't see how this check helps against any potentially nil pointers.

Reply:
To appear in `-oyaml` sure.. but I don't think this affected the go type.

Reply:
Like does this resolve any panics?